qa/suites/fs/upgrade/nofs/tasks/1-upgrade.yaml

```diff
@@ -39,7 +38,7 @@ tasks:
     - ceph versions
     - ceph osd dump -f json-pretty
     - ceph fs dump
-    - ceph osd require-osd-release quincy
+    - ceph osd require-osd-release squid
```

Comment:
@batrick wont removing branch from `install.upgrade` lead to issues here?

Reply:
If you were backporting this, possibly. But this PR should not be.

Reply:
okay, then we're good to go; I've incorporated your suggestion @batrick, PTAL.